focus.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "FOCUS",
+  "aliases": [
+    "Fisheries Open Source Community Software"
+  ],
+  "description": "FOCUS is an open-source community that aims to oversee and maintain free software dedicated to the management and preservation of the fishery resource.",
+  "website": "http://www.focus.fish/about/",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "http://www.focus.fish/"
```

Comment:
@Msgl could you update the license to point to this explicit [link](https://github.com/UnionVMS/UVMS-MobileTerminalModule-APP/blob/dev/LICENSE). Thanks.